docs/functions/functions.json

```diff
@@ -437,6 +437,77 @@
 				true
 			]
 		},
+		{
+			"name": "max",
+			"id": "max",
+			"unary": false,
+			"category": "math",
+			"role": "function",
+			"description": "Calculate the maximum of all numbers in the list.",
+			"argument": [
+				{
+					"name": "list",
+					"optional": false,
+					"type": "List of numbers",
+					"description": "The numbers you are trying to find the max of."
+				}
+			],
+			"returnType": "Number",
+			"returnDescription": "The minimum number"
+		},
+		{
+			"name": "round",
+			"id": "round",
+			"unary": false,
+			"category": "math",
+			"role": "function",
+			"description": "Round a given number to the specified decimal place. Round to whole number if decimal place not specified.",
```

Comment:
This sounds a little too technical and less intuitive too me. 
I'd write something like "Rounds a number to the nearest integer, or to the nearest number with a given number of decimals if the `decimals` argument is used." but that's just off the top of my head, I'm sure there's a far better way to phrase it.